### PR TITLE
frontend: Remove GUI for Track 2

### PR DESCRIPTION
**Pull request by kyoto**
Remove GUI for Track 2.

Removes all frontend code and backend API code and the associated tests.

~~Does _**not**_ include removing any Go packages that may now be unused.~~

Restoring the GUI will require migrating to the YAML Tectonic config file format. Some of that work was done in #2884.

cc: @sym3tri, @robszumski, @squat, @cpanato

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -28,29 +28,6 @@ temporarily merged into the target branch of the pull request.
...
-## Running GUI tests on PRs
-
-The GUI tests include integration tests for the AWS and the Baremetal GUI
-installer.
-
-### Actions required
-- Add the `run-gui-tests` GitHub label
-
-### FAQ
-- *I am not able to add labels, what should I do?*
```
Comment (squat) [inline tests/README.md]: Why eliminate this block?

Comment (kyoto) [reply tests/README.md]: These FAQs are actually repeated in other sections of the doc.

**Comment by kyoto** [comment]
ok to test

**Comment by squat** [comment]
@kyoto can we add another commit that re-vendors? I can do this for you if you like, but we should really eliminate all unused vendored packages as part of this

**Comment by kyoto** [comment]
Rebased

**Comment by alexsomesan** [comment]
@squat @kyoto 
Is this change still in need of a followup remove of the remaining unused code? I'd like to merge soonish to be able to depend on the frontend not being there.

**Comment by squat** [comment]
@alexsomesan: @kyoto is taking some time off. I’m working on cleaning up the last remaining vendor files. Should be done tomorrow.

**Comment by squat** [comment]
@mxinden @enxebre @alexsomesan PTAL. All unneeded vendored go code is now removed.

**Comment by cpanato** [comment]
@squat need to remove the `bazel test installer/frontend:unit --test_output=all` from jenkinsfile

File: installer/README.md [hunk 2]
```diff
@@ -29,52 +14,14 @@ Get a [license](https://account.coreos.com) and follow the guides to create Tect
 
 - [Go 1.8](https://golang.org/doc/install)
 - [Nodejs >=8.x](https://nodejs.org/en/download/)
```
Comment (enxebre) [inline installer/README.md]: we can remove Nodejs as well , can't we?

**Comment by enxebre** [comment]
This will need cleanup as well https://github.com/kyoto/tectonic-installer/blob/disable-gui-tests/images/tectonic-builder/Dockerfile#L48 actually are we using this image anymore cc @cpanato @mxinden

**Comment by cpanato** [comment]
and the node as well https://github.com/kyoto/tectonic-installer/blob/disable-gui-tests/images/tectonic-builder/Dockerfile#L66

**Comment by squat** [comment]
ok to test

**Comment by squat** [comment]
ok to test

**Comment by enxebre** [comment]
All the nodejs related stuff in bazel WORKSPACE can go away https://github.com/kyoto/tectonic-installer/blob/disable-gui-tests/WORKSPACE#L21

**Comment by enxebre** [comment]
This can be removed https://github.com/kyoto/tectonic-installer/blob/disable-gui-tests/bill-of-materials.json#L363

**Comment by enxebre** [comment]
this wont be needed anymore https://github.com/kyoto/tectonic-installer/blob/disable-gui-tests/tests/smoke/bare-metal/packet/main.tf#L119 but could be removed as a part of follow up for keep removing obsolete bits

**Comment by enxebre** [comment]
This can be removed https://github.com/kyoto/tectonic-installer/blob/disable-gui-tests/installer/scripts/license-gen/collect_project_licenses.sh#L174

**Comment by squat** [comment]
The last two things are unrelated to the GUI so let’s do that separately. I’ll regenerate the BOM and fix the WORKSPACE. Thanks for finding those.

**Comment by mxinden** [comment]
restarted aws tests

**Pull request merged**